apis/Google.Cloud.Language.V1/Google.Cloud.Language.V1.IntegrationTests/SmokeTest.cs

```diff
@@ -0,0 +1,54 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Cloud.ClientTesting;
+using Grpc.Core;
+using Xunit;
+
+namespace Google.Cloud.Language.V1.IntegrationTests
+{
+    public sealed class SmokeTest : CloudProjectFixtureBase
+    {
+        [Fact]
+        public void AnalyzeSentiment_Language()
```

Comment:
It's hard to tell what the `_Language` means here - I'd change it to `_DefaultLanguage` to make it clearer.